lib/Infer/ExhaustiveSynthesis.cpp

```diff
@@ -32,6 +32,12 @@ namespace {
     "The larger the number is, the more fine-grained debug "
     "information will be printed"),
     cl::init(0));
+  static cl::opt<unsigned> NumInstructions("souper-exhaustive-synthesis-num-instructions",
+    cl::desc("Number of instruction of RHS (default=1). "),
+    cl::init(1));
+  static cl::opt<bool> DisableBigQuery("souper-exhaustive-synthesis-disable-big-query",
```

Comment:
let's flip this around: have a flag (default=false) for enabling big query, not for disabling it